.konflux/manifests/application.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Application
+metadata:
+  name: serverless-operator-konflux
+spec:
+  description: serverless-operator (konflux banch)
```

Comment:
Nit: banch -> branch

Reply:
Good catch. I'll update it

Reply:
updated: 4e93335a059916d81a3313c7348b8a8ff95503a9